### DISCUSSION TITLE
Use MMM Data Platform

### DISCUSSION THREAD

**Discussion by JonasSchroederOtto** [google/meridian#185] [Q&A]
I very much like the idea of adding GQV and other data to the MMM. 

I tried to follow the guide here https://developers.google.com/meridian/docs/basics/using-mmm-data-platform
However, I have no idea where I could find the Data Platform. Is this something upcoming and will be linked to the guide once it's there? Or do I need to look for my organization's owner of Google Ad accounts?

Best,
Jonas

**Comment by matthewfallshaw** [comment]
I also don't have access to the MMM Data Platform.
I do have access to all of the Github Meridian resources via my Github account (including the ability to post this!)

My invitation email included:
> STEP 3 - Request MMM data: To enable Meridian’s new methodology innovations with richer MMM data, please visit [Google’s MMM Data Platform](https://partnerdash.google.com/apps/mmm-frontend) and submit a request for core Google media inputs (impressions, clicks, costs), Google Query Volume, and Reach & Frequency data for your brand.

I enabled "Partner Dash" via Google Admin Console > Apps > Additional Google services > Partner Dash, and see 403 "doesn't have permission" errors at the https://partnerdash.google.com/apps/mmm-frontend URL.

Things that seem unlikely to be, but *might* be relevant:
1. My Github account has multiple email addresses attached to it, including the company email address I was invited under, but that email is not my "primary" Github email.

**Comment by jonquil2002** [comment]
@JonasSchroederOtto @matthewfallshaw  Thank you for your feedback and sorry to hear that the directions for accessing the MMM Data Platform are confusing.

Each organization has a unique instance of the MMM Data Platform landing page, which is why it isn't provided in the documentation. Please contact your organization's primary user of the MMM Data Center so that they can enable your access and provide you with the link to your organization's landing page.

Although we cannot link these unique landing pages to the guide or emails, I'll update the [Use MMM Data Platform](https://developers.google.com/meridian/docs/basics/using-mmm-data-platform) documentation to make this information clearer.
